index.html

```diff
@@ -1,12 +1,476 @@
+                                galley of type."
+                                </p>
+                            </div>
+                            <div class="column-img">
+                                <img src="./img/user.png" />
+    
+                            </div>
+                            <div class="column-full-name">
+                                <h3 class="name">Mark Todd</h3>
+                            </div>
+                            <div class="column-job">
+                                <p>General Counsel and CCO</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                
+            </div>
+            
+            
+        </section>
+
+        <section class="contact-us section-margin">
+            <div class="container">
+                <div class="content">
+                    <div class="flex">
+
+                        <div class="column">
+                            <div class="column-img">
+                                <img src="img/illustration-contact-us.svg" />
+                            </div>          
+                        </div>
+
+                        <div class = "column flex-column">
+                            <div class="column-title">
+                                <h2>Contact Us</h2>
+                            </div>
+                            <div class="column-desc">
+                                <p>We believe in the power of conversations. Start one with us today!</p>
+                            </div>
+
+                            <div class="form">
+
+                                <div class="form-flex">
+
+                                    <div class="form-column flex-column ">
+                                        <div class="column-field">
+                                            <input class="field" type="text" placeholder="Input name"> </input>
+                                        </div>
+                                        <div class="column-field">
+                                            <input class ="field" type="text" placeholder="Zip code"> </input>
+                                        </div>
+                                    </div>
+                                    <div class="form-column flex-column ">
+                                        <div class="column-field">
+                                            <input class="field" placeholder="Email"> </input>
+                                        </div>
+                                        <div class="column-field">
+                                            <input class="field" placeholder="Company"> </input>
+                                        </div>
+                                    </div>
+                                    
+                                </div>
+
+                                <div class="form-column-mesg">
```

Comment:
shortened words could cause issues and negatively affect team-work by requiring additional from other developers in the case when they need to work on your components after you.